src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -337,11 +337,11 @@ internal static MethodSymbol FindContainingGenericMethod(Symbol symbol)
             return null;
         }
 
-        private static RefKind GetModifiers(SyntaxTokenList modifiers, out SyntaxToken argPassingKeyword, out SyntaxToken paramsKeyword, out SyntaxToken thisKeyword)
+        private static RefKind GetModifiers(SyntaxTokenList modifiers, out SyntaxToken refOrOutKeyword, out SyntaxToken paramsKeyword, out SyntaxToken thisKeyword)
```

Comment:
I think this helper is ok as is.

---
In reply to: [103793818](https://github.com/dotnet/roslyn/pull/17469#discussion_r103793818) [](ancestors = 103793818,103585079)